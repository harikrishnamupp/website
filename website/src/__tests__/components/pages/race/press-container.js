import React from 'react'
import renderer from 'react-test-renderer'
import PressContainer from '~components/pages/race/press-container'

describe('Components : Race : Press Container', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <PressContainer>
          <span>this is a child</span>
        </PressContainer>,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
