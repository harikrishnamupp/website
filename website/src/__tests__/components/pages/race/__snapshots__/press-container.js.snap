// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Race : Press Container renders correctly 1`] = `
<div
  className="container"
>
  <span>
    this is a child
  </span>
</div>
`;
